### PR TITLE
feat/server

### PR DESCRIPTION
**Pull request by BhanuPratap-Singh-Naruka**
created a server using fastify

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
```
Comment (abhimishra01) [inline server/package.json:4]: lets add a small description for our server

**Review by abhimishra01 — CHANGES_REQUESTED** [review]

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "main": "index.js",
```
Comment (abhimishra01) [inline server/package.json:5]: entry point is :-
```
main: server.js 
```

Comment (abhimishra01) [reply server/package.json:5]: sry
``` server.ts```

File: server/package.json (hunk 1 above)
Comment (BhanuPratap-Singh-Naruka) [reply server/package.json]: done

File: server/package.json (hunk 2 above)
Comment (BhanuPratap-Singh-Naruka) [reply server/package.json]: done

File: server/server.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import {config} from "dotenv"
```
Comment (abhimishra01) [inline server/server.ts:1]: prettier formatting seems off, we will setup prettier in upcoming PRs

**Review by abhimishra01 — APPROVED** [review]

**Pull request merged**